Validation/MtdValidation/plugins/MtdTracksValidation.cc

```diff
@@ -989,6 +1008,38 @@ void MtdTracksValidation::bookHistograms(DQMStore::IBooker& ibook, edm::Run cons
   meTrackPathLenghtvsEta_ = ibook.bookProfile(
       "TrackPathLenghtvsEta", "MTD Track pathlength vs MTD track Eta;|#eta|;Pathlength", 100, 0, 3.2, 100.0, 400.0, "S");
 
+  meTrackSigmaTof_[0] =
+      ibook.book1D("TrackSigmaTof_Pion", "Sigma(TOF) for pion hypothesis; #sigma_{t0} [ps]", 100, 0, 50);
```

Comment:
@noepalm at least for the pion hypothesis 50 ps appears redundant, lots of unused bins (i.e. memory). Kaons are in an intermediate situation, while the chosen range makes sense for protons. Probably you may reduce the range and have a smaller bin granularity (i.e. 1 bin/ps).

Why are there many underflows in these histograms, looking at the test output DQM file from the PR tests like https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-2491ca/37307/runTheMatrix-results/24834.0_TTbar_14TeV+2026D98/DQM_V0001_R000000001__Global__CMSSW_X_Y_Z__RECO.root ?

Reply:
answering to myself: tracks with no time assigned, right? The initialisation value is -1 in the ```TrackExtenderWithMTD```

Reply:
I checked in a small TTbar sample and all negative values came from the -1 initialization, so that seems to be the case.

I've now also updated the binning for pions to 10 bins in the [0, 5] ps range and for kaons to 25 bins in [0, 25] ps.